src/components/Listing/Listing.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <div class="advertiser-details">
+    <h1 class="title">{{ listing["listing"]["title"] }}</h1>
+    <!-- <span>{{ listing["listing"]["adress"]["locationId"] }}</span> -->
```

Comment:
Why this line is commented ? 
Can you exclude ?

Reply:
I will use it when refining the component's style.
I can uncomment, it's an information that I want to show.